### PR TITLE
Feature/feature/momza 1096 adding missing translations

### PR DESCRIPTION
**Pull request by OMosimege**
In this PR, we add missing translations for postbirth Auto Responses for Moms switching SMS/WA

### CODE REVIEW

File: changes/tasks.py [hunk 1]
```diff
@@ -548,8 +554,54 @@ def switch_channel(self, change):
...
+                    "MomConnect msgs for kids aged 1-2 are only on WA. "
+                    "To stop msgs, reply 'STOP' (std rates apply)"
+                )
+
+            if reason == "whatsapp_contact_check_fail":
+                utils.ms_client.create_outbound(
+                    {
+                        "to_identity": change.registrant_id,
+                        "content": fail_contact_check_text,
+                        "channel": "WHATSAPP",
```
Comment (rudigiesler) [inline changes/tasks.py]: If it's a whatsapp contact check fail, that means that they're not on WhatsApp, so we can't send this message over WhatsApp, so we probably want to send this message over SMS. The same with the unsent event.

Comment (OMosimege) [reply changes/tasks.py]: thanks for picking this up, I missed it

File: changes/tasks.py [hunk 2]
```diff
@@ -548,8 +554,54 @@ def switch_channel(self, change):
...
+                        },
+                    }
+                )
+
+            elif reason == "postbirth_wa_unsent_event":
+                utils.ms_client.create_outbound(
+                    {
+                        "to_identity": change.registrant_id,
+                        "content": unsent_event_text,
+                        "channel": "sms",
```
Comment (rudigiesler) [inline changes/tasks.py]: I think that the channel name here is `JUNE_TEXT`.

Also, we won't need the template metadata for SMS.

Comment (OMosimege) [reply changes/tasks.py]: ahh I see, thank you

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**